python/GafferUI/CompoundEditor.py

```diff
@@ -1528,10 +1528,10 @@ def editorKeyPress( editor, event ) :
 		if not isinstance( editor, GafferUI.NodeSetEditor ) :
 			return False
 
-		if event.key == "N" :
+		if event.key == "N" and event.modifiers is None :
```

Comment:
Have you tested this? I would expect it to have broken the editor shortcuts. `is None` is only ever going to be true for an object of type `None`, but `event.modifiers` has type `Modifiers`. One of the enum values is named "None", but that is different to Python's built in `None` object. `not e.modifiers` is probably what you really want.

Reply:
It does work, but will change

Reply:
Of course. I'd double checked it had made `<ctrl>-n` work again, and missed the obvious problem... sorry about that! Not the best start to the day 🙃. Fixed in https://github.com/GafferHQ/gaffer/pull/3548/commits/0e6ad73eb2696c9d6707ee2e10852aefd9b52e08